routes/index.js

```diff
@@ -1,235 +1,242 @@
 const express = require("express");
 const pug = require("pug");
 const fs = require("fs");
+const { Level } = require("level");
+const db = new Level("./db", { valueEncoding: "json" });
 
 const router = express.Router();
 
 const shaderDirectory = "./public/shaders";
 
 let accounts = [];
+loadState();
 
-const names = [
```

Comment:
removed these cuz we can now have persistent accounts i.e. no need to recreate each time we make changes